tools/Makefile.am

```diff
@@ -57,3 +57,20 @@ lou_translate_SOURCES =	lou_translate.c
 lou_trace_SOURCES = lou_trace.c
 lou_compare_SOURCES = lou_compare.c
 lou_checkyaml_SOURCES =	lou_checkyaml.c
+
+if !CROSS_COMPILING
```

Comment:
Not sure if this is *strictly* correct. We could still have someone that has all the necessary tools installed. Just because we are cross compiling doesn't mean the tools will not be installed on the target systems (even patgen might be installed :-))

Reply:
OK first of all we're assuming CROSS_COMPILING means we are cross compiling for Windows, which is already incorrect, but we do the same [in tests/Makefile.am](https://github.com/liblouis/liblouis/blob/1b5166b046b5d6f5701f1ca0a752f0882a125b42/tests/Makefile.am#L103).

If someone has indeed cross-compiled for Windows, the chance that he has all the dependencies installed is very small, that's why I thought it's better to ignore that case because in all the other cases it will cause confusion.

The required dependencies are:
- bash
- (GNU) make
- patgen
- python3

I'm not sure what happens if someone has all these dependencies installed somehow and manages to run the tool. I don't think it's very likely that it will actually work.

Reply:
Hm, OK, you're right we indeed do the same in `tests/Makefile.am`. 

> If someone has indeed cross-compiled for Windows, the chance that he has all the dependencies installed is very small

Makes sense. Let's leave it as it is